lighthouse-core/test/lib/dependency-graph/simulator/tcp-connection-test.js

```diff
@@ -5,7 +5,6 @@
  */
 'use strict';
 
-// eslint-disable-next-line
```

Comment:
here and in `network-analyzer-test.js` were fixed in #9006 but the new `--fix` missed these because of these disable comments. I assume these were originally disabled for `max-len` but maybe moving files reduced the path length over time. It's fine without the disable now